index.html

```diff
@@ -107,11 +112,69 @@ <h3><a href="https://github.com/FIRSTMap/firstmap.github.io">View or edit FIRSTM
             </div>
         </div>
 
+        <div id="search">
+            <h2 class="stripe">Search</h2>
+
+            <div id="searchSection" hidden>
+                <h3>Search</h3>
+                <div id="teamSearch">
+                    <input type="search" id="teamSearchInput" onkeyup="inputKeyUp(this)" />
+                    <button onclick="search()">Search</button>
+                </div>
+
+                <div id="eventSearch" hidden>
+                    <input type="search" id="eventSearchInput" />
+                    
+                    <ul id="eventSearchList">
+                    </ul>
+                </div>
+
+                <select id="searchType" onchange="(() => { updateSearchType(); })()">
+                    <option value="team">Team</option>
+                    <option value="event">Event</option>
+                </select>
+                <br />
+                <h6 id="searchInfoText"></h6>
+            </div>
+
+            <div id="filterSection" hidden>
```

Comment:
Nit: we don't use camelCase for IDs/classes anywhere else as far as I can see—we generally use hyphenated names so I'd prefer consistency with that